File: week 1/3052/miso.py
#C++로 작성한 코드 
"""#include <iostream>
#include <string>
#include <vector>
#include<algorithm>
using namespace std;

int main() {
	vector<int> v(10);
	
	for (int i = 0; i < 10; i++) {
		cin >> v[i] ;
	}
	for (int i = 0; i < 10; i++) {
		v[i] = v[i] % 42;
	}

	sort(v.begin(), v.end());
	int p = 1;
	for (int i = 0; i < 9; i++) {
		if (v[i] != v[i + 1])
			p++;

	}
	cout << p;

}"""

#파이썬 코드
a=set()
for i in range(10):
    num= int(input())
    a.add(num%42)

print(len(a))

File: week 2 /2908/miso.cpp
#include <iostream>
#include<vector>
#include<string>

using namespace std;

/* 


*/

int main()
{
    string s1, s2,c1,c2;
    cin>>s1>>s2;
    
    for(int i=2;i>=0;i-- ){
        c1+=s1[i];
        c2+=s2[i];
    }
    
    int a, b;
    a=stoi(c1);
    b=stoi(c2);
    
    cout<<max(a,b);
    
    
}
